tests/de/assist_satellite_HassBroadcast.yaml

```diff
@@ -0,0 +1,23 @@
+language: "de"
+tests:
+  - sentences:
+      - "gib bekannt das abendessen ist fertig"
+      - "gib durch das abendessen ist fertig"
+      - "gebe bekannt das abendessen ist fertig"
+      - "gebe durch das abendessen ist fertig"
+      - "lass alle wissen das abendessen ist fertig"
+      - "lass alle hören das abendessen ist fertig"
+      - "verkünde das abendessen ist fertig"
+      - "verkünde überall das abendessen ist fertig"
+      - "sag überall das abendessen ist fertig"
+      - "sage überall das abendessen ist fertig"
+      - "sage überall Bescheid das abendessen ist fertig"
+      - "sage überall, dass das Abendessen fertig ist"
+      - "verkünde überall, dass das Abendessen fertig ist"
+      - "gib Bescheid, dass das Abendessen fertig ist"
+      - "gib überall Bescheid, dass das Abendessen fertig ist"
```

Comment:
these will produce a different response: "das Abendessen fertig ist" instead of "das Abendessen ist fertig". that's the transformation issue I mentioned. I see no way to work around that unless we want to introduce a long list of verbs and conjugations of "sein", "haben", "werden"....

Reply:
ahhh ... with response you mean the broadcasted message?

Reply:
Sorry, yes. That was unnecessarily ambiguous

Reply:
than we should remove the `dass` at all - so only allow things like "gib überall Bescheid das Abendessen ist fertig"

Reply:
This issue must be the same for at least some other languages (IIRC dutch should have the same issue). If we don't come up with a solution on our own, we can improve later on, when HassBroadcast is implemented in those languages.

Reply:
> than we should remove the `dass` at all - so only allow things like "gib überall Bescheid das Abendessen ist fertig"

but then asking "gib überall bescheid dass das abendessen fertig ist" will result in a broadcast "dass das abendessen fertig ist". we can only chose pest ("das abendessen fertig ist") or cholera ("dass das abendessen fertig ist") I guess. both sound weird.

Reply:
tbh, I don't think the original approach in EN to simply speak out {message} is that good. I'd rather have a broadcast like "Hiermit wird verkündet: {message}".

Reply:
> tbh, I don't think the original approach in EN to simply speak out {message} is that good. I'd rather have a broadcast like "Hiermit wird verkündet: {message}".

this would need a change i the core I think. So for now i would just remove the `dass` and all related test sentences. The user will recognize, that the message is replayed as is, there is no transformation or similar in between.

Reply:
yes, also a chime before the announcement like in https://github.com/nimroddolev/chime_tts would be great

while the message is simply repeated, I agree with your idea of removing `dass` as it's the less weird behaviour.

Reply:
> yes, also a chime before the announcement like in https://github.com/nimroddolev/chime_tts would be great

even this is a change in the core, and maybe also in the firmware of the VPE itself :thinking: but nothing for this PR for now :wink: